programme-schedule.md

```diff
@@ -0,0 +1,56 @@
+# Week 1: Induction
+
```

Comment:
@damienomurchu @JameelB yes we chatted about this also yesterday. The context of this repo is content for students. We could have our own guide internally.